tensorflow_datasets/core/api_utils.py

```diff
@@ -45,16 +67,24 @@ def disallow_positional_args(wrapped=None, allowed=None):
     return functools.partial(disallow_positional_args, allowed=allowed)
 
   @wrapt.decorator
-  def disallow_positional_args_dec(fn, instance, args, kwargs):
+  def disallow_positional_args_dec(fn: Callable[[], Any],
```

Comment:
The function can take arbitrary number of argument, so should be either `fn: Callable[..., Any]`, or `FnT` (better, as indicate intent the signature is the same as the top level function).

Same think below

Reply:
I have put Callable[..., Any]